src/main/java/ai/deepcode/javaclient/DeepCodeRestApiImpl.java

```diff
@@ -118,24 +118,28 @@ private interface CreateBundleCall {
     @retrofit2.http.Headers("Content-Type: application/json")
     @POST("bundle")
     Call<CreateBundleResponse> doCreateBundle(
-      @Header("Session-Token") String token, @Body FileContentRequest files);
+      @Header("Session-Token") String token,
+      @Header("snyk-org-name") String orgName,
```

Comment:
What's happening if the org is empty? Question applies to all calls, I think.

Reply:
it's fine. in this case default org will be taken into account